apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -96,7 +97,8 @@ paths() ->
         "/clients/:clientid/subscribe/bulk",
         "/clients/:clientid/unsubscribe",
         "/clients/:clientid/unsubscribe/bulk",
-        "/clients/:clientid/keepalive"
+        "/clients/:clientid/keepalive",
+        "/sessions_count"
```

Comment:
Q: shouldn't this be namespaced under `/clients` too?

Reply:
I think this deserves a root path.